fe/fe-core/src/main/java/org/apache/doris/load/DeleteHandler.java

```diff
@@ -709,6 +745,51 @@ public void write(DataOutput out) throws IOException {
 
     public static DeleteHandler read(DataInput in) throws IOException {
         String json = Text.readString(in);
-        return GsonUtils.GSON.fromJson(json, DeleteHandler.class);
+        DeleteHandler deleteHandler = GsonUtils.GSON.fromJson(json, DeleteHandler.class);
+        deleteHandler.removeOldDeleteInfos();
```

Comment:
Hi, there is problem in here. When we are load image, we shouldn't write the editLog, or else replayRemoveOldDeleteInfos() method will triggered after loading image immediately.